poetry.lock

```diff
@@ -6,6 +6,20 @@ category = "main"
 optional = true
 python-versions = "*"
```

Comment:
not sure this is meant to be committed (maybe I'm wrong)

Reply:
Yes, the lock file should be committed.